Automattic/wp-calypso
# Subscriptions Management: Enable `comments-list-controls` and `sites-list-controls` feature flags

Closes https://github.com/Automattic/wp-calypso/issues/76141.

⚠️ Please do not merge until we are ready to launch the searching and sorting on "Sites" and "Comments" tab to EN users.

## Proposed Changes

* enable the `subscription-management/sites-list-controls` feature flag in production (and in all preceding environments)
* enable the `subscription-management/comments-list-controls` feature flag in production (and in all preceding environments)
* make sure to launch to EN users only

## Testing Instructions

The changes related to the feature flags will be visible only once this PR is merged and deployed. Prior merging, we can review the code, make sure it doesn't introduce any regressions and check whether the search and sort features are displayed to EN users only:

1. Check out and build the PR locally.
2. Apply the correct `subkey` cookie value to your `calypso.localhost:3000` host.
3. Navigate to http://calypso.localhost:3000/subscriptions/sites as an external user with English locale. → The search and sorting features should be visible and working as expected. The same should apply to those features on the http://calypso.localhost:3000/subscriptions/comments page.
4. Navigate to http://calypso.localhost:3000/subscriptions/sites/it (or use any other Mag-16 locale). → The search and sorting features should not be available. The same should apply to those features on the http://calypso.localhost:3000/subscriptions/comments page.

## Pre-merge Checklist
